test/FakeDataProducerDAQModule.cpp

```diff
@@ -29,6 +29,11 @@ FakeDataProducerDAQModule::FakeDataProducerDAQModule(const std::string& name)
   , thread_(std::bind(&FakeDataProducerDAQModule::do_work, this, std::placeholders::_1))
   , outputQueue_(nullptr)
   , queueTimeout_(100)
+  , nIntsPerVector_(999)
+  , starting_int_(-999)
+  , ending_int_(-999)
+  , wait_between_sends_ms_(999)
```

Comment:
Should we move initializers for concrete types to the header, using brace syntax?

Reply:
After Slack discussion, agreed that parameters which are only ever meant to be settable via JSON should be set in the header.